group_vars/all.yml

```diff
@@ -207,7 +207,7 @@ vm_image_url:
    rhel8: http://download.hosts.prod.upshift.rdu2.redhat.com/released/RHEL-8/8.1.0/BaseOS/x86_64/images/rhel-guest-image-8.1-263.x86_64.qcow2
    rhel8_2: http://download.eng.pek2.redhat.com/released/RHEL-8/8.2.0/BaseOS/x86_64/images/rhel-guest-image-8.2-290.x86_64.qcow2
 #Neutron backend, if not set default will be used.
-#neutron_backend: ovn
+neutron_backend: ovn
```

Comment:
Add a comment that we are not supporting ml2/ovs for DVR right now because it needs changes in roles_data.yaml file.

Reply:
ack